clients/go/pkg/auditopt/config.go

```diff
@@ -123,8 +123,8 @@ func interceptorFromConfigFile(ctx context.Context, path string, lookuper envcon
 			return fmt.Errorf("SecurityContext must be provided to use interceptor")
 		}
 
-		if cfg.RequireJustification {
-			jvsClient, err := client.NewJVSClient(ctx, &client.JVSConfig{JVSEndpoint: cfg.JVSEndpoint})
+		if cfg.Justification != nil && cfg.Justification.Enabled {
+			jvsClient, err := client.NewJVSClient(ctx, &client.JVSConfig{JVSEndpoint: cfg.Justification.JVSPublicKeysEndpoint})
```

Comment:
We now should move this to `clientFromConfig` func since it's no longer gRPC specific?

Reply:
I will do the change once I work on the https://github.com/abcxyz/lumberjack/issues/283

Reply:
Ack